.github/workflows/deploy.yaml

```diff
@@ -26,12 +28,16 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
+          flavor:
+            # don't automatically tag with `latest`; we do this conditionally in the `tags` section
+            latest=false
           images: |
             ${{ vars.DOCKERHUB_REPO }}
             ${{ vars.GAR_LOCATION }}-docker.pkg.dev/${{ vars.GCP_PROJECT_ID }}/${{ vars.GAR_REPOSITORY}}/autograph
           tags: |
-            type=semver,pattern={{raw}}
-            type=raw,value=latest,enable={{is_default_branch}}
+            type=semver,pattern={{raw}},enable=${{ github.event_name == 'release' && github.event.action == 'released' }}
+            type=semver,pattern={{raw}}-prerelease,enable=${{ github.event_name == 'release' && github.event.action == 'prereleased' }}
```

Comment:
Can we call this `stage` and `prod` for easier mapping to our deploy environments? Or maybe @jbuck would have knowledge of what other folks are doing for CI/CD consistency

Reply:
Sure, I really have no preference what it's called.